### PR TITLE
153460466 track an activity

### PR DESCRIPTION
**Pull request by Andrej1198**
# Problem this pull request is addressing
This pull request is giving the sample app the ability to have an activity log that can track a single activity.

# Associated Pivotal Story
https://www.pivotaltracker.com/n/projects/1907103/stories/153460466

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -7,5 +7,24 @@
 
   get 'users/index'
   resources :users
+
+
+#  scope 'users/:id' do
+#    post '/activities/new', to: 'activities#create'
+#    resources :activities
+#  end
+
```
Comment (mZhang0125) [inline config/routes.rb]: should remove comments

File: app/views/activities/new.html.erb [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+  <p>
+    <%= form.label :"Quantity" %><br>
+    <%= form.text_field :quantity %>
+  </p>
+
+  <p>
+    <%= form.label :"Date" %><br>
+    <%= form.text_field :date %>
+  </p>
```
Comment (mZhang0125) [inline app/views/activities/new.html.erb]: better to use form.date_field

Comment (Andrej1198) [reply app/views/activities/new.html.erb]: I didn't know that was an available option, thanks!

File: config/routes.rb (hunk 1 above)
Comment (Andrej1198) [reply config/routes.rb]: Good catch, I'll search the entire document for other like this

**Pull request merged**